BUILD.bazel

```diff
@@ -35,15 +36,29 @@ pkg_tar(
     },
 )
 
+# When built with --stamp, creates a non-deterministic output file for pushing images to a remote registry.
+# With --nostamp, produces a deterministic output so dependents get cache hits.
+expand_template(
+    name = "deb_version_file",
+    out = "_deb_version.txt",
+    stamp_substitutions = {"_TAG_": "0.0.0~{{BUILD_SCM_TIMESTAMP}}-{{BUILD_SCM_REVISION}}"},
```

Comment:
What are your thoughts on dropping the 0.0.0 prefix while we're at it? A BUILD_SCM_TIMESTAMP also happens to be a valid Debian version number, as it ends up doing a simple lexicographic comparison.

Reply:
Of course, even better.